docs/contribution/folder-structure.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
This file is not correct for this repository. I don't think you need this file and this document entry for pgm-io. The folder structure is pretty simple.

Reply:
this happens when you multitask too much 😬 i thought i deleted this. i will go over this PR once again to check whether everything is correct